### DISCUSSION TITLE
Valutazione

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#651] [Richieste di aiuto] [answered]
Buonasera
Se volessi rendere i voti non visibili alla famiglia in media (cioè il docente sceglie se è si o no) come dovrei fare?
Quale sarebbe il codice?
Grazie.

**Comment by trinko** [comment]
E' già possibile inserire un voto che non sia visibile e un voto che, pur essendo visibile, non entra nella media.

**Reply by Davide031** [reply]
Ma se voglio che fa media se il docente clicca su si?

**Comment by trinko** [comment]
Ah, ricordo ora che questa discussione è stata già fatta... [vedi](https://github.com/iisgiua/giuaschool/discussions/583)

Non ripeto quanto ho già detto: questo possibilità un registro elettronico non la deve permettere.

**Reply by Davide031** [reply, marked as answer]
Va bene grazie lo stesso
